script.js

```diff
@@ -19,14 +19,140 @@ function hamburgerAction() {
   }
 }
 
-function scrollAction() {
-  if (hamburger.style.display === 'block') {
-    mainContainer.style.position = 'relative';
-    hamburger.style.display = 'none';
-    burgerMenu.classList.replace('nav-menu-click', 'nav-menu');
-    burgerImg.src = 'img/icon-menu.png';
-  }
-}
+// function scrollAction() {
+//   if (hamburger.style.display === 'block') {
+//     mainContainer.style.position = 'relative';
+//     hamburger.style.display = 'none';
+//     burgerMenu.classList.replace('nav-menu-click', 'nav-menu');
+//     burgerImg.src = 'img/icon-menu.png';
+//   }
+// }
 
 menuIcon.addEventListener('click', hamburgerAction);
-menuList.addEventListener('click', scrollAction);
\ No newline at end of file
+// menuList.addEventListener('click', scrollAction);
```

Comment:
- [ ] Commented codes are supposed to be removed from the codebase to leave it clean and neat 😎 please remove them